### ISSUE TITLE
es4x scripts and package bin uses /bin/sh instead of /bin/bash which breaks for lack of pushd

### ISSUE THREAD

**Issue by abner** [reactiverse/es4x#92]
es4x scripts and package bin uses /bin/sh instead of /bin/bash which breaks for lack of pushd.

Since ubuntu 17, the default bash in Ubuntu is `dash`, which lacks pushd and does not support all the sintax supported by bash.

I found this errors when running `npm install` or `npm start` on an example project using es4x-pm.

```
node_modules/.bin/es4x: pushd: not found  /node_modules/.bin/es4x: 6: node_modules/.bin/es4x: Bad substitution
```

I just replaced the first line in es4x binary to `#!/bin/bash` and in the project generated binary also and the errors were fixed.

**Comment by abner** [comment]
Closing because i saw it was already fixed on develop branch

**Comment by abner** [comment]
any chance to have this published to npm in a 0.7.3 minor version?
